pkgs/applications/graphics/img-cat/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "imgcat";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "trashhalo";
+    repo = "imgcat";
+    rev = "v${version}";
+    sha256 = "0a8i0q2f109dljc9pcxvg2csaz2h04m0mdidjcwdzg6x6ly6skbg";
+  };
+
+  vendorSha256 = "03dg7xz9vd81wx0h0igsnl3n6x5lvxkjg4ynjrxxcz3dy65w7da2";
+
+  doCheck = false;
```

Comment:
A comment as to why tests are disabled would be nice.

Reply:
There is no tests in upstream, you can remove doCheck.

Reply:
Good catch, thanks! I've removed `doCheck`